addons/website_slides/controllers/main.py

```diff
@@ -186,20 +186,24 @@ def channel(self, channel, category=None, tag=None, page=1, slide_type=None, sor
 
     @http.route(['/slides/channel/add'], type='http', auth='user', methods=['POST'], website=True)
     def slide_channel_create(self, *args, **kw):
+        channel = request.env['slide.channel'].create(self._slide_channel_create_prepare_values(kw))
```

Comment:
Why is taking the slide_channel_create args that you don't handle ? Just keep kwargs.